### ISSUE TITLE
PetabImporter: Read in configuration files

### ISSUE THREAD

**Issue by yannikschaelte** [ICB-DCM/pyPESTO#117]
For a start, introduce a configuration file (ini format?) that is not mandatory to the PeTab format, but which the petabimporter can read in and translate to amici and pypesto options.

The specifics and language of such a file can be revised later. Criteria: Human-readable, easy to edit, and easy to read in from different languages (mainly python, R, matlab).

This is needed in particular for serializing the model creation.

**Comment by dweindl** [comment]
I vote for [YAML](https://en.wikipedia.org/wiki/YAML) and could also live with less readable JSON. I think specifying lists and dictionaries in .ini is no fun.

**Comment by yannikschaelte** [comment]
agreed, ini is just for simple one-value options. yaml looks nice. python integration not that good apparently, but should be no problem using pyYAML.

what would be a possible use case for needing a list or dict? 
possibly the OptimizerOptions? these could be realized in ini also using sections.

**Comment by dweindl** [comment]
This is only loosely related to PEtab. Renaming.

**Comment by dweindl** [comment]
@yannikschaelte Any progress or concrete plans here?

**Comment by yannikschaelte** [comment]
No, unfortunately not, except for this _slightly_ outdated branch https://github.com/ICB-DCM/pyPESTO/tree/feature_config_file
